src/main/java/bot/command/instruction/concrete/MarkNotDoneInstruction.java

```diff
@@ -0,0 +1,26 @@
+package bot.command.instruction.concrete;
+
+import bot.Storage;
```

Comment:
Good and consistent grouping of the imports (from different semantical layers), throughout all your classes.